meta: add efiop as a maintainer

efiop opened the pull request:
I've noticed that there are multiple update PRs not getting any activity at all and I'm proposing to add me as a maintainer to help out. We are using this package in dvc and our releases are blocked because of the lack of updates to this project :slightly_frowning_face: 

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

efiop committed:
meta: add efiop as a maintainer

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * noarch: python recipes are required to have a lower bound on the python version. Typically this means putting `python >=3.6` in **both** `host` and `run` but you should check upstream for the package's Python compatibility.

efiop committed:
meta: add lower bound for python

Per https://github.com/conda-forge/flatten-dict-feedstock/pull/4#issuecomment-901184165

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

efiop committed:
meta: bump build number

efiop commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.12, and conda-forge-pinning 2021.08.18.12.46.46

efiop commented:
@conda-forge-admin @bgruening @conda-forge/help-python (hope I didn't ping someone inappropriately) Hey guys, friendly ping about this. From https://github.com/conda-forge/flatten-dict-feedstock/pull/1:

This project hasn't been merging any updates since May (no PRs merged at all here) and we are blocked by this in https://github.com/conda-forge/dvc-feedstock/pull/229 If someone can add me as a maintainer here (by merging this PR) - I will be happy to sort out pending updates here. :pray:

efiop commented:
@mariusvniekerk Thank you so much! :pray:
